src/magic_folder/testing/web.py

```diff
@@ -10,11 +10,16 @@
 """
 Test-helpers for clients that use the WebUI.
 
-NOTE: this is code copied from Tahoe-LAFS until there is a release
-after 1.14.0 that we can depend on. Once 1.15.0 or later is release,
-this code can be deleted.
+NOTE: This code should be in upstream Tahoe-LAFS.  None of it exists in
+1.14.0.  Some of it has been pushed upstream and will make it into 1.15.0
+without further efforts but other parts have not.  Changes here should always
+be pushed upstream quickly.
```

Comment:
Is there a corresponding PR for tahoe?

Reply:
Yea okay I guess I contradicted myself here, didn't I.  No, there's no PR yet.  I'm not sure what I was thinking when I made this doc change.  I think ... trying to defend against divergence that would be a pain to reconcile.  Maybe given the pace of Tahoe-LAFS development and the currently active set of developers that's an unreasonable fear though?

Reply:
I updated this to be internally consistent, at least.  If it still doesn't seem like the right plan then we can discuss further after this PR.